package.json

```diff
@@ -84,12 +81,7 @@
             {
                 "command": "o.restore",
```

Comment:
Any reason not to lose the "o." prefix here? Maybe "dotnet."? I don't think it has any user impact, so feel free to ignore me. :smile: